c2corg_api/views/user.py

```diff
@@ -108,8 +111,9 @@ def __init__(self, request):
         partial(validate_unique_attribute, "username")])
     def post(self):
         user = schema_create_user.objectify(self.request.validated)
+        user.forum_username = user.username  # same usernames by default
         user.password = self.request.validated['password']
-        user.update_validation_nonce(VALIDATION_EXPIRE_DAYS)
+        user.update_validation_nonce('regemail', VALIDATION_EXPIRE_DAYS)
```

Comment:
We also use constants everywhere in the code. Notably in the very same line. ;)

In this case it would be helpful to see at one place which values can be used as `purpose`.